asapdiscovery-data/asapdiscovery/data/services_config.py

```diff
@@ -0,0 +1,30 @@
+from pathlib import Path
+from typing import Optional
+
+from pydantic import BaseSettings, validator
+
+
+class PosteraSettings(BaseSettings):
+    POSTERA_API_KEY: str
+    POSTERA_API_URL: str = "https://api.asap.postera.ai"
+    POSTERA_API_VERSION: str = "v1"
+
+
+class S3Settings(BaseSettings):
+    AWS_ACCESS_KEY_ID: str
+    AWS_SECRET_ACCESS_KEY: str
+    BUCKET_NAME: str
+    BUCKET_PREFIX: Optional[str] = None
+
+
+class CloudfrontSettings(BaseSettings):
+    CLOUDFRONT_DOMAIN: str
+    CLOUDFRONT_KEY_ID: str
+    CLOUDFRONT_PRIVATE_KEY_PEM: str
```

Comment:
Consider adding fields so we can have a description of what should go here this is the only one that is not clear by name alone to me.